locale/ja/Notepad2.rc

```diff
@@ -1587,17 +1587,17 @@ END
 
 IDD_AUTOSAVE DIALOGEX 0, 0, 184, 81
 STYLE DS_SETFONT | DS_MODALFRAME | DS_FIXEDSYS | WS_POPUP | WS_CAPTION | WS_SYSMENU
-CAPTION "AutoSave Settings"
+CAPTION "自動保存の設定"
 FONT 8, "MS Shell Dlg", 0, 0, 0x0
 BEGIN
-    AUTOCHECKBOX    "Enable AutoSave.",IDC_AUTOSAVE_ENABLE,7,7,170,10,WS_GROUP | WS_TABSTOP
-    LTEXT           "AutoSave period (in second):",IDC_STATIC,14,19,120,10,SS_RIGHT
+    AUTOCHECKBOX    "自動保存を有効化。" AutoSave.",IDC_AUTOSAVE_ENABLE,7,7,170,10,WS_GROUP | WS_TABSTOP

```

Comment:
`AutoSave."` need to be removed to fix build error:

https://github.com/zufuliu/notepad2/runs/4838639542?check_suite_focus=true#step:5:210
> Notepad2.rc(1593): error RC2104: undefined keyword or key name: AutoSave. [D:\a\notepad2\notepad2\locale\ja\Notepad2(ja).vcxproj]